andela/bad_ass-ah-frontend
# #164489907 track comment history

**Description of Task to be completed?**

- User should be able to track edit comment history

**How should this be manually tested?**

- Ensure that a user is connected

- From the homepage click on a given article

- Post one comment

- Edit the comment

- Then an `Edited` button will appear, click on that and a `modal` of edited comment history will be displayed

**What are the relevant pivotal tracker stories?**

[#164489907](https://www.pivotaltracker.com/story/show/164489907)

**Screenshots (if appropriate)**
![image](https://user-images.githubusercontent.com/46047244/59556127-c2b8e980-8fbd-11e9-85ef-020e68492b0a.png)

![image](https://user-images.githubusercontent.com/46047244/59556143-e67c2f80-8fbd-11e9-894d-451413a15293.png)
